pippy/PipelineDriver.py

```diff
@@ -1217,6 +1281,28 @@ def wait_for_confirmation(n):
         return node
 
 
+class run_until_criteria:
```

Comment:
Make this private i.e. `_run_until_criteria`

Reply:
Thanks, fixed.